src/forms/labels/input-label.js

```diff
@@ -55,34 +57,50 @@ const propTypes = {
   label: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
   name: PropTypes.string.isRequired,
   tooltip: PropTypes.string,
+  required: PropTypes.bool,
+  requiredIndicator: PropTypes.oneOfType([ PropTypes.string, PropTypes.bool ]),
   ...togglePropTypes('tooltipShown')
 }
 
 const defaultProps = {
   hint: '',
   label: '',
   tooltip: '',
+  requiredIndicator: '*'
```

Comment:
Technically not necessary since undefined is falsey but since the `required` prop is not required, it's best practice to explicitly provide a default here (which would be false)

Reply:
That's a good point, since we are providing defaults for props not required within the component, but in my mind that prop technically does have a default from the `input` element? The question is can that be lost on someone just looking at this component?

Reply:
If the `input` element is providing a default, then should we be able to make this prop required?